docs/src/modules/components/ComponentsApiContent.js

```diff
@@ -158,24 +135,17 @@ export default function ComponentsApiContent(props) {
 
     const componentNameKebabCase = kebabCase(componentName);
 
-    const useNamedImports = source.startsWith('@mui/base');
-
-    const subpathImport = useNamedImports
-      ? `import { ${namedImportName} } from '${defaultImportPath}';`
-      : `import ${defaultImportName} from '${defaultImportPath}';`;
-
-    const rootImport = `import { ${namedImportName} } from '${namedImportPath}';`;
-
-    const importInstructions = `${subpathImport}
-// ${t('or')}
-${rootImport}`;
-
     return (
       <React.Fragment key={`component-api-${key}`}>
         <MarkdownElement>
           <Heading hash={componentNameKebabCase} text={`${componentName} API`} />
           <Heading text="import" hash={`${componentNameKebabCase}-import`} level="h3" />
-          <HighlightedCode code={importInstructions} language="jsx" />
+          <HighlightedCode
+            code={imports.join(`
```

Comment:
Nit: could you extract this to a variable? Having multiline template string within JSX looks a bit off to me.